compliance_checker/acdd.py

```diff
@@ -226,8 +226,8 @@ def check_lat_extents(self, ds):
         if not (hasattr(ds, 'geospatial_lat_min') and hasattr(ds, 'geospatial_lat_max')):
             return
 
-        lat_min = ds.geospatial_lat_min
-        lat_max = ds.geospatial_lat_max
+        lat_min = float(ds.geospatial_lat_min)
+        lat_max = float(ds.geospatial_lat_max)
```

Comment:
Could you add exception handlers to these float casts?  Sometimes there are values that can't be cast to float.

Reply:
i.e. `except ValueError:`